python/dataproc_templates/jdbc/README.md

```diff
@@ -375,4 +375,102 @@ export JARS="gs://my-gcp-proj/jars/mysql-connector-java-8.0.29.jar,gs://my-gcp-p
+               JDBC.BIGQUERY.TEMP.BUCKET.NAME
+               --jdbc.bigquery.input.url JDBC.BIGQUERY.INPUT.URL
+               --jdbc.bigquery.input.driver
+               JDBC.BIGQUERY.INPUT.DRIVER
+               --jdbc.bigquery.input.table JDBC.BIGQUERY.INPUT.TABLE
+               [--jdbc.bigquery.input.partitioncolumn JDBC.BIGQUERY.INPUT.PARTITIONCOLUMN]
+               [--jdbc.bigquery.input.lowerbound JDBC.BIGQUERY.INPUT.LOWERBOUND]
+               [--jdbc.bgiquery.input.upperbound JDBC.BGIQUERY.INPUT.UPPERBOUND]
+               [--jdbc.bigquery.numpartitions JDBC.BIGQUERY.NUMPARTITIONS]
+               [--jdbc.bigquery.output.mode {overwrite,append,ignore,errorifexists}]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --jdbc.bigquery.output.dataset JDBC.BIGQUERY.OUTPUT.DATASET
+                        BigQuery dataset for the output table
+  --jdbc.bigquery.output.table JDBC.BIGQUERY.OUTPUT.TABLE
+                        BigQuery output table name
+  --jdbc.bigquery.temp.bucket.name JDBC.BIGQUERY.TEMP.BUCKET.NAME
+                        Spark BigQuery connector temporary bucket
+  --jdbc.bigquery.input.url JDBC.BIGQUERY.INPUT.URL
+                        JDBC input URL
+  --jdbc.bigquery.input.driver JDBC.BIGQUERY.INPUT.DRIVER
+                        JDBC input driver name
+  --jdbc.bigquery.input.table JDBC.BIGQUERY.INPUT.TABLE
+                        JDBC input table name
+  --jdbc.bigquery.input.partitioncolumn JDBC.BIGQUERY.INPUT.PARTITIONCOLUMN
+                        JDBC input table partition column name
+  --jdbc.bigquery.input.lowerbound JDBC.BIGQUERY.INPUT.LOWERBOUND
+                        JDBC input table partition column lower
+                        bound which is used to decide the partition
+                        stride
+  --jdbc.bgiquery.input.upperbound JDBC.BGIQUERY.INPUT.UPPERBOUND
+                        JDBC input table partition column upper
+                        bound which is used to decide the partition
+                        stride
+  --jdbc.bigquery.numpartitions JDBC.BIGQUERY.NUMPARTITIONS
+                        The maximum number of partitions that can be
+                        used for parallelism in table reading and
+                        writing. Default set to 10
+  --jdbc.bigquery.output.mode {overwrite,append,ignore,errorifexists}
+                        Output write mode (one of:
+                        append,overwrite,ignore,errorifexists)
+                        (Defaults to append)
+
+```
+
+## General execution: 
+
+```
+export GCP_PROJECT=<gcp-project-id> 
+export REGION=<region>  
+export GCS_STAGING_LOCATION=<gcs staging location> 
+export SUBNET=<subnet>   
+export JARS="<gcs_path_to_jdbc_jar_files>/mysql-connector-java-8.0.29.jar,<gcs_path_to_jdbc_jar_files>/postgresql-42.2.6.jar,<gcs_path_to_jdbc_jar_files>/mssql-jdbc-6.4.0.jre8.jar,<gcs_path_to_jdbc_jar_files>/spark-bigquery-latest_2.12.jar"
```

Comment:
Can BQ-Spark connector be directly referenced from its gcs location ?
gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar

Reply:
Yes, it can. Ref: https://cloud.google.com/dataproc/docs/tutorials/bigquery-connector-spark-example